common/views/components/IIIFViewer/IIIFViewer.js

```diff
@@ -0,0 +1,339 @@
+// @flow
+import styled from 'styled-components';
+import { classNames, spacing, font } from '@weco/common/utils/classnames';
+import NextLink from 'next/link';
+import { itemUrl } from '@weco/common/services/catalogue/urls';
+import {
+  type IIIFCanvas,
+  type IIIFImageService,
+} from '@weco/common/model/iiif';
+import Paginator, {
+  type PaginatorRenderFunctionProps,
+  type PropsWithoutRenderFunction as PaginatorPropsWithoutRenderFunction,
+} from '@weco/common/views/components/RenderlessPaginator/RenderlessPaginator';
+import Control from '@weco/common/views/components/Buttons/Control/Control';
+import IIIFResponsiveImage from '@weco/common/views/components/IIIFResponsiveImage/IIIFResponsiveImage';
+
+const IIIFViewerPaginatorButtons = styled.div.attrs(props => ({
+  className: classNames({
+    'flex absolute flex--h-center': true,
```

Comment:
How do you feel about standardising how we do this?

Reply:
I've lost track of where the chewing the code agendas are, but I thought that that was an item for whenever we next meet (tomorrow)? In short, yes, I'm all for standardising!